widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -137,4 +141,23 @@ public static void clearVisitorSession() {
         Dependencies.getControllerFactory().destroyControllers();
         Glia.clearVisitorSession();
     }
+
+    private static final Consumer<GliaException> updateVisitorExceptionConsumer = exception -> {
+        if (exception != null) {
+            Logger.e(TAG, "Visitor Info update failed: " + exception.toString());
+        } else {
+            Logger.e(TAG, "Visitor Info updated Success");
+        }
```

Comment:
Wanted to confirm, is there some reason we are not notifying the integrator of the request result? I think additionally to this we can also let the integrator register his own callback if he wants to know if the request was made successfully?

Reply:
as I understand it was necessary to add only a possibility without implementation.

Reply:
You mean to add the callback to the `VisitorInfoUpdateRequest.Builder` ?

I think that there should be some way to notify the integrator of the request was successful or not. And having a callback in the update method is the simplest solution. Though having it in the builder is also good.
@dukhovnyi